src/main/java/com/google/changestreams/sample/bigquery/changelog/fullschema/SchemaUtils.java

```diff
@@ -0,0 +1,539 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.changestreams.sample.bigquery.changelog.fullschema;
+
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.changestreams.sample.bigquery.changelog.fullschema.model.SpannerColumn;
+import com.google.changestreams.sample.bigquery.changelog.fullschema.model.SpannerTable;
+import com.google.cloud.ByteArray;
+import com.google.cloud.Date;
+import com.google.cloud.Timestamp;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import com.google.cloud.spanner.*;
+import org.json.JSONObject;
+
+import java.util.*;
+
+public class SchemaUtils {
+
```

Comment:
Does this class need to only expose static methods? It seems all methods use the `databaseClient`. Could we transform this class to receive a database client in the constructor and have an instance instead?

Reply:
But then how can we test it with the mocked client? Maybe have a setClient method just for testing?

Reply:
You'd need to inject the SchemaUtils into the classes that use it, like:

```
final SchemaUtils schemaUtils = new SchemaUtils(databaseClient);
final MyClass myClass = new MyClass(schemaUtils);

...
public class MyClass {
  public MyClass(SchemaUtils schemaUtils) {
    this.schemaUtils = schemaUtils;
  }
}
```

Reply:
Fixed.